src/java.base/share/classes/java/io/InputStreamReader.java

```diff
@@ -48,7 +48,7 @@
  *
```

Comment:
Oddly, none of the reference in this class to the default charset are links to Charset.defaultCharset().
That would be a useful addition, either in the class javadoc or in the 1-arg constructor that uses the default charset.

Reply:
Thanks, Roger. Both are good suggestions. Will incorporate them into the next iteration.